src/labels.jl

```diff
@@ -3,9 +3,9 @@
 ##############
 immutable StateLabel{N}
     label::NTuple{N}
-    hash::Uint64
-    StateLabel(label::NTuple{N}) = new(label, hash(label))
-    StateLabel(label::NTuple{N}, h::Uint64) = new(label, h)
+    hash::UInt64
+    StateLabel{N}(label::NTuple{N}) where N = new(label, hash(label))
+    StateLabel{N}(label::NTuple{N}, h::UInt64) where N = new(label, h)
```

Comment:
Some more `UInt64` --> `UInt` situations. It seems there are a bunch of these - they should all get fixed, but I'll just stop commenting on them.